keps/prod-readiness/sig-apps/2879.yaml

```diff
@@ -2,4 +2,6 @@ kep-number: 2879
 alpha:
   approver: "@wojtek-t"
 beta:
-  approver: "@wojtek-t"
\ No newline at end of file
+  approver: "@wojtek-t"
+stable:
```

Comment:
In addition to comments by @jeremyrickard in:
https://github.com/kubernetes/enhancements/pull/4237#issuecomment-1742262859

I have some more:

> Are there any tests for feature enablement/disablement?

Can you link them?


2. High-level question : the additional load on apiserver is potentially concerning [in GA we will no longer be able to disable the feature].
Do you have any rate-limitting (e.g. we don't update a job more than once per second or sth like that)?

If so - can you please describe. If not, we definitely should add something.

Reply:
Thanks for the comments @wojtek-t . There are unit and integration tests, now linked. 

As for the rate-limiting we have a mechanism, batching all Job reconciliations. The `syncJob` invocations are batched by time of 1s, and deduplicated as we use the `AddAfter` to queue the syncs: https://github.com/kubernetes/kubernetes/blob/e8abe1af8dcb36f65ef7aa7135d4664b3db90e89/pkg/controller/job/job_controller.go#L560.